test/test_autotokenizer.py

```diff
@@ -3,6 +3,7 @@
 import unittest
 
 import numpy as np
+import ftfy
```

Comment:
why do you need import it here? I think it was used by HF tokenizers.

Reply:
yeah but they did not check in that PR so HF natively does not import it so it fails without that import locally - might still pass on the ci but it will cause issues for locally testing in the future so i think it is better to have it in the code.

Reply:
not quite follow here, If you want other be aware that ftfy should be installed to to avoid the test failure. You may check ftfy installation here and raise a warning it is missed.